### DISCUSSION TITLE
Astroid has been approved on JED

### DISCUSSION THREAD

**Discussion by sonvnn** [templaza/astroid-framework#749] [Announcements]
Hi Astroiders

We are happy to announce that Astroid has been approved on JED as a testament to our seriousness and constant efforts to make Astroid become a useful and free product for the community.

If you are a fan of Astroid. Please help me rate/review Astroid now. Your contribution will help us bring Astroid to more people and be known by more people.

### TO RATE/REVIEW FOR ASTROID
Go to JED site: https://extensions.joomla.org/extension/style-a-design/templating/astroid-framework/
Login to your account and switch to Reviews tab and click to "Write a review" button

Thank you very much!
With my best regards,
Sonny

**Comment by Chacapamac** [comment]
GREAT NEWS! — I prepare my review and invite EVERYBODY using Astroid to do the same to spread the news that the best framework for Joomla is available and recognized. 

**UPDATE** — I cannot leave reviews on the JED????????? — Anybody experience that. When I press “Submit” I go back to Astroid extension page and no acknowledgment that my review was sent?????? — I get that problem for ages and complaint to the JED but no luck????????

**Reply by vtheod** [reply]
First they check your review and then they publish it. Just wait ;)

**Reply by Chacapamac** [reply]
Yep, it finally work, already **6 FIVE STARS REVIEWS**, LET’S FILL THAT BABY UP! ⭐️⭐️⭐️⭐️⭐️

**Comment by Dnsftl** [comment]
Review posted.
